components/AdminUserDashboard.tsx

```diff
@@ -13,6 +13,12 @@ export default function AdminUserDashboard(): JSX.Element {
     push,
     query: { dashboard }
   } = useRouter()
+  const { API_MANAGER_ENABLED } = process.env
+  const activeUserTypes = USER_TYPES.filter((userType) => {
```

Comment:
This needs to be implemented on the user management page `manage.tsx` too.

Reply:
I think the admin user should always be able to see users even if the module is disabled (to resolve any bugs or issues that may occur). I was considering a check of number of users per type, but that would involve moving some requests around. What do you think?

Reply:
I don't think that makes sense. If we introduce API users for testing, we should remove them when we do the final deployments.

Reply:
Disabled tabs only shown after unhiding them in 4a6dc1ca06e5ff1db397c10ff8cc8cb63187b35b